fizz__buzz.rb

```diff
@@ -0,0 +1,30 @@
+puts"enter the value of n"
+n=gets.chomp.to_i
```

Comment:
hi,
for automated testing we require you write you code within the function that was already created in the template you cloned